src/lib/wagmi.ts

```diff
@@ -1,7 +1,7 @@
 // Custom connectors
+import { createFunkitWagmiConfig } from '@funkit/connect';
 import type { PrivyClientConfig } from '@privy-io/react-auth';
-import { createConfig } from '@privy-io/wagmi';
```

Comment:
will we lose custom privy logic that they shim into createConfig. Might be good idea to ask privy guys